lang/cs/Org.Apache.REEF.Common/metrics/Api/IMetricsCollector.cs

```diff
@@ -43,5 +44,16 @@ public interface IMetricsCollector
         /// <param name="info">Meta-data info of the record.</param>
         /// <returns>Record builder for the record.</returns>
         IMetricsRecordBuilder CreateRecord(IMetricsInfo info);
+
+        /// <summary>
+        /// Clears up the metrics collector. Removes all record builders and records.
+        /// </summary>
+        void Clear();
```

Comment:
This makes the metrics collector mutable. What's the rationale for that? And what happens in the distributed setting if only a subset of the Tasks call `.Clear()`